test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/AichatwebTemplatesTests.cs

```diff
@@ -42,6 +45,17 @@ public AichatwebTemplatesTests(ITestOutputHelper log)
 
     [Fact]
     public async Task BasicTest()
+    {
+        await TestTemplateCoreAsync(scenarioName: "Basic");
+    }
+
+    [Fact]
+    public async Task BasicAspireTest()
+    {
+        await TestTemplateCoreAsync(scenarioName: "BasicAspire", templateArgs: ["-as"]);
+    }
```

Comment:
Thanks for setting this up to have the scenario name pivot!

Reply:
The testing infra makes this easeeey ;)